datamodels/2.x/itop-config/dictionaries/pt_br.dict.itop-config.php

```diff
@@ -22,17 +22,17 @@
  */
 Dict::Add('PT BR', 'Brazilian', 'Brazilian', array(
 
-	'Menu:ConfigEditor' => 'Configuration~~',
-	'config-edit-title' => 'Configuration File Editor~~',
-	'config-edit-intro' => 'Be very cautious when editing the configuration file.~~',
-	'config-apply' => 'Apply~~',
-	'config-apply-title' => 'Apply (Ctrl+S)~~',
-	'config-cancel' => 'Reset~~',
-	'config-saved' => 'Successfully recorded.~~',
-	'config-confirm-cancel' => 'Your changes will be lost.~~',
-	'config-no-change' => 'No change: the file has been left unchanged.~~',
-	'config-reverted' => 'The configuration has been reverted.~~',
-	'config-parse-error' => 'Line %2$d: %1$s.<br/>The file has NOT been updated.~~',
-	'config-current-line' => 'Editing line: %1$s~~',
-	'config-saved-warning-db-password' => 'Successfully recorded, but the backup won\'t work due to unsupported characters in the database password.~~',
+	'Menu:ConfigEditor' => 'Configuração',
+	'config-edit-title' => 'Editor do arquivo de configuração',
+	'config-edit-intro' => 'Tenha cuidado ao editar o arquivo de configuração.',
+	'config-apply' => 'Aplicar',
+	'config-apply-title' => 'Aplicar (Ctrl+S)',
+	'config-cancel' => 'Resetar',
+	'config-saved' => 'Gravado com sucesso.',
+	'config-confirm-cancel' => 'Suas mudanças serão perdidas.',
+	'config-no-change' => 'Sem alteração: o arquivo ficou sem alteração.',
+	'config-reverted' => 'A configuração foi restaurada.',
+	'config-parse-error' => 'Linha %2$d: %1$s.<br/>O arquivo não foi atualizado.',
+	'config-current-line' => 'Editando linha: %1$s~~',
```

Comment:
```suggestion
	'config-current-line' => 'Editando linha: %1$s',
```

'~~' needs to be removed

Reply:
Fixed